overlord/snapstate/snapmgr.go

```diff
@@ -139,6 +139,11 @@ type SnapState struct {
 	// InstanceKey is set by the user during installation and differs for
 	// each instance of given snap
 	InstanceKey string `json:"instance-key,omitempty"`
+
+	// RefreshPostponedTime records the time when the refresh was first
+	// attempted but postponed for because the snap was busy. This value is
```

Comment:
Extra for I think in this sentence: "attempted but postponed <s>for</s> because the snap was busy."